python_modules/libraries/dagster-dg/dagster_dg/cli/list.py

```diff
@@ -104,64 +104,64 @@ def _pretty_features(obj: PackageObjectSnap) -> Text:
     return text
 
 
-def _package_object_table(entries: Sequence[PackageObjectSnap]) -> Table:
+def _plugin_object_table(entries: Sequence[PluginObjectSnap]) -> Table:
     sorted_entries = sorted(entries, key=lambda x: x.key.to_typename())
     table = DagsterInnerTable(["Symbol", "Summary", "Features"])
     for entry in sorted_entries:
         table.add_row(entry.key.to_typename(), entry.summary, _pretty_features(entry))
     return table
 
 
-def _all_packages_object_table(
-    registry: RemotePackageRegistry, name_only: bool, feature: Optional[PackageObjectFeature]
+def _all_plugins_object_table(
+    registry: RemotePluginRegistry, name_only: bool, feature: Optional[PluginObjectFeature]
 ) -> Table:
     table = DagsterOuterTable(["Package"] if name_only else ["Package", "Objects"])
```

Comment:
Should this be updated to "Plugin"?

Note that that's a slightly annoying change because `test_list_commands.py` hardcodes the expected output (we should really convert those to snapshot tests to make our lives easier)